test/DummySolverInterface.c

```diff
@@ -13,17 +13,22 @@ const char *fake_data_name;
 double *fake_bounding_box;
 double *fake_coordinates;
 
-void precicec_createSolverInterface(const char *participantName,
-                                    const char *configurationFileName,
-                                    int solverProcessIndex,
-                                    int solverProcessSize)
+void precicec_createParticipant_withCommunicator(const char *participantName,
+                                                 const char *configurationFileName,
+                                                 int solverProcessIndex,
+                                                 int solverProcessSize,
+                                                 void *communicator)
 {
-    fake_read_write_buffer = malloc(SIZE * fake_dimensions * sizeof(double));
     fake_dimensions = 3;
```

Comment:
For the sake of consistency, lets define a `fake_data_dimensions` and a `fake_mesh_dimensions` here and use it appropriately in the fake API.